app/presenters/notices/setup/prepare-return-forms.presenter.js

```diff
@@ -9,6 +9,21 @@ const NotifyAddressPresenter = require('./notify-address.presenter.js')
 const { formatLongDate } = require('../../base.presenter.js')
 const { naldAreaCodes, returnRequirementFrequencies } = require('../../../lib/static-lookups.lib.js')
 
+const RETURN_TYPE = {
+  week: {
+    columns: 2,
+    columnSize: 14
```

Comment:
This can be called 'rows' if we can clearly illustrate that a row is not across the page but confined to the column. So a page with 2 columns would have two rows 14 vertically. 

The data is top down and not across.